Examples/instance-flow.md

```diff
@@ -223,10 +276,13 @@ else
 fi;
 ```
 
-### Terminate the Instance
+## Terminate the Instance
```

Comment:
Deleting is not the same as terminating. Terminate runs the stop and terminate events scripts, and then removes the cloud resources, but does not remove the instance from the CAM DB. This steps involves sending first a `DELETE` with an `operation=terminate` URL parameter, which terminates the instance, followed by a `DELETE` with an `operation=delete` parameter.